api/include/opentelemetry/trace/tracer.h

```diff
@@ -85,6 +85,19 @@ class Tracer
     return this->StartSpan(name, attributes, {}, options);
   }
 
+  template <class T,
+            nostd::enable_if_t<common::detail::is_key_value_iterable<T>::value> * = nullptr>
+  nostd::shared_ptr<Span> StartSpan(
+      nostd::string_view name,
+      std::initializer_list<std::pair<nostd::string_view, common::AttributeValue>> attributes,
+      const StartSpanOptions &options = {}) noexcept
+  {
+    return this->StartSpan(name,
+                           nostd::span<const std::pair<nostd::string_view, common::AttributeValue>>{
+                               attributes.begin(), attributes.end()},
+                           {}, options);
+  }
+
```

Comment:
Why is this method templatized?

Reply:
From what I remember - I had run into needing this for Mac OS X clang only, for `examples/http/server.cc` below.

Reply:
Ok. but the template parameter `T` is not used anywhere inside the function definition. Does it really need to be a template function or normal function will do?

Reply:
Let me try it locally again before I revert this..

Reply:
You are right. That was a left-over from the other debug effort I had to do, to ensure that the types we pass to `absl::variant` are actually well-defined in that variant. The other fix I did in `server.cc` was sufficient. I removed the unused template.